<ui version="4.0" >
 <class>BusSelectorDialog</class>
 <widget class="QWidget" name="BusSelectorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bus Selector</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="trackComboBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Capture Buses</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="busesListWidget" />
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Channels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBoth" >
          <property name="text" >
           <string>Both</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioLeftOnly" >
          <property name="text" >
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRightOnly" >
          <property name="text" >
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Playback Buses</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="busesListWidgetPlayback" />
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Channels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBothPlayback" >
          <property name="text" >
           <string>Both</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioLeftOnlyPlayback" >
          <property name="text" >
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRightOnlyPlayback" >
          <property name="text" >
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
